www/src/pages/docs/cli/integrations/css.md

```diff
@@ -431,6 +444,122 @@ The sky is the limit with mode selectors, but some popular patterns are:
 
 :::
 
+### Color Scheme
+
+The CSS plugin supports automatically generating the [`color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme) property to help browsers optimize rendering for light and dark themes. This feature works with both the base selector and individual mode selectors.
```

Comment:
```suggestion
The CSS plugin supports automatically generating the [`color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme) property to help browsers respect users' prefered color schemes. This feature works with both the base selector and individual mode selectors.
```

nit: I don't think theme is the right word here, and mentioning user preferences would help root what color schemes are for juniors who are still learning vocabulary.